fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/tiering/source/TieringSource.java

```diff
@@ -95,6 +106,13 @@ public SourceReader<TableBucketWriteResult<WriteResult>, TieringSplit> createRea
         return new TieringSourceReader<>(sourceReaderContext, flussConf, lakeTieringFactory);
     }
 
+    private static byte[] generateOperatorHash() {
```

Comment:
/** This follows the operator uid hash generation logic of flink {@link StreamGraphHasherV2}. */